toxav/toxav.h

```diff
@@ -246,9 +246,9 @@ typedef enum Toxav_Err_Call {
  * receiving are both enabled by default.
  *
  * @param friend_number The friend number of the friend that should be called.
- * @param audio_bit_rate Audio bit rate in Kb/sec. Set this to 0 to disable
+ * @param audio_bit_rate Audio bit rate in kbit/sec. Set this to 0 to disable
  *   audio sending.
- * @param video_bit_rate Video bit rate in Kb/sec. Set this to 0 to disable
+ * @param video_bit_rate Video bit rate in Mbit/sec. Set this to 0 to disable
```

Comment:
Why lowercase 'k' but upper case 'M'? Any reason for them not to be the same case?

Reply:
Hm, looking around, it looks like the lowercase 'k' and uppercase 'M' are customary letter cases so I guess that's fine.

https://en.m.wikipedia.org/wiki/Data-rate_units